spec/spec_helper.rb

```diff
@@ -1,14 +1,6 @@
 require 'support/database_cleaner.rb'
 require 'devise'
```

Comment:
you shouldn't need to require this here. Maybe there's a specific file to require to get the test helpers, but you shouldn't need to require the whole gem.